frontend/src/components/Header/components/Nav/index.tsx

```diff
@@ -87,5 +88,88 @@ export default function Nav({ className, pathname }: Props): JSX.Element {
         />
       </LinkWrapper>
     </NavWrapper>
+  ) : (
+    <NavWrapper className={className}>
+      <LinkWrapper>
+        <Link href={ROUTES.COLLECTIONS} passHref>
+          <AnchorButton
+            active={isRouteActive(pathname, ROUTES.COLLECTIONS)}
+            data-testid="collections-link"
+            href="passHref"
+            minimal
+            onClick={() => {
+              track(EVENTS.COLLECTIONS_CLICK_NAV);
+            }}
+            text="Collections"
+          />
+        </Link>
+      </LinkWrapper>
+      <LinkWrapper>
+        <Link href={ROUTES.DATASETS} passHref>
+          <AnchorButton
+            active={isRouteActive(pathname, ROUTES.DATASETS)}
+            href="passHref"
+            minimal
+            onClick={() => {
+              track(EVENTS.DATASETS_CLICK_NAV);
+            }}
+            text="Datasets"
+          />
+        </Link>
+      </LinkWrapper>
+      <LinkWrapper>
+        <Link href={ROUTES.WHERE_IS_MY_GENE} passHref>
+          <AnchorButton
+            active={isRouteActive(pathname, ROUTES.WHERE_IS_MY_GENE)}
+            href="passHref"
+            minimal
+            onClick={() => {
+              track(EVENTS.WMG_CLICK_NAV);
+            }}
+            text="Gene Expression"
+          />
+        </Link>
+      </LinkWrapper>
+      <LinkWrapper>
+        <Link href={ROUTES.CELL_GUIDE} passHref>
+          <AnchorButton
+            active={isRouteActive(pathname, ROUTES.CELL_GUIDE)}
+            href="passHref"
+            minimal
+            onClick={() => {
+              track(EVENTS.CELL_GUIDE_CLICK_NAV);
+            }}
+            text="Cell Guide"
+          />
+        </Link>
+        <BetaChip label="Beta" size="small" />
+      </LinkWrapper>
+      <NavDivider />
+      <LinkWrapper>
+        <AnchorButton
+          href={CENSUS_LINK}
+          minimal
+          onClick={() => {
+            track(EVENTS.CENSUS_CLICK_NAV);
+          }}
+          rel="noopener"
+          target="_self"
+          text="Documentation"
+        />
+      </LinkWrapper>
+      <LinkWrapper>
+        <Link href={ROUTES.CENSUS_DIRECTORY} passHref>
+          <AnchorButton
+            active={isRouteActive(pathname, ROUTES.CENSUS_DIRECTORY)}
+            href="passHref"
+            minimal
+            onClick={() => {
+              // TODO ANALYTICS
+            }}
+            text="Directory"
+          />
+        </Link>
+      </LinkWrapper>
```

Comment:
if this is the only addition, should we just `isCensusDirectory  && <LinkWrapper> .... </LinkWrapper>` instead of a giant ternary?

Reply:
ternary is just a feature flag for now, I'm going to be including more changes behind the ternary in a future PR:
![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/8716829/7a719789-dffb-4740-b487-fb89841db002)

Reply:
Ahh got it! Yeah that looks quite different from the current one. And since we're touching the top nav, we will also need to think about updating Explorer and Census Docs page, since they're all in different repos!